dht/dht.go

```diff
@@ -1,14 +1,17 @@
 package dht
 
 import (
-	"fmt"
-	"io"
+	"math/rand"
 	"sort"
 	"sync"
 
 	"github.com/renproject/aw/wire"
 	"github.com/renproject/id"
-	"github.com/renproject/surge"
+)
+
+var (
+	// DefaultSubnet is used to refer to all known signatories.
+	DefaultSubnet = id.Hash{}
```

Comment:
Instead of explicitly supporting the `DefaultSubnet` in `package dht`, it should be `package gossip` that understands the `DefaultSubnet` and opts to call `Addrs(n)` instead of `Subnet(subnet)`. This keeps the DHT simple, and `package gossip` is already doing this kind of multiplexing.